benchmarks/guid/comptime-guid-parse-bench.zig

```diff
@@ -0,0 +1,14 @@
+const std = @import("std");
+const bench = @import("root");
+
+pub fn setup(gpa: std.mem.Allocator, options: *bench.Options) ![]const u8 {
+    _ = gpa;
+    options.useChildProcess();
+    return options.zig_exe;
+}
+
+pub fn run(gpa: std.mem.Allocator, zig_exe: []const u8) !void {
+    return bench.exec(gpa, &[_][]const u8{
+        zig_exe, "build-exe", "comptime-guid-parse.zig",
```

Comment:
This will avoid doing some unnecessary things that could skew the timings:

```suggestion
        zig_exe, "build-obj", "comptime-guid-parse.zig",
```